src/main/java/com/tvtracker/enterprise/dao/MediaEntryDAO.java

```diff
@@ -66,37 +52,15 @@ public MediaEntry fetch(int id) throws SQLException, IOException, ClassNotFoundE
      */
     @Override
     public boolean delete(int id) throws SQLException, IOException, ClassNotFoundException {
-        addWhere("id", id);
-        return delete();
+        allMediaEntries.remove(id);
+        return true;
     }
 
-    /**
-     * Method for updating a MediaEntry record in the database
-     *
-     * @param mediaEntry MediaEntry object to be used for updating a database record
-     * @return boolean indicating a successful update
-     */
     @Override
     public boolean update(MediaEntry mediaEntry) throws SQLException, IOException, ClassNotFoundException {
-        setAllColumnValues(mediaEntry);
-        addWhere("id", mediaEntry.getId());
-        return update();
+        return false;
```

Comment:
Lots of logic removed here... and returning false all of the time.  It looks like this should have been left as it were.